tests/unit/memefficiency.tcl

```diff
@@ -397,6 +405,17 @@ run_solo {defrag} {
                 if {!$::no_latency} {
                     assert {$max_latency <= 30}
                 }
+
+                if {$type eq "standalone"} {
+                    r publish chan "hello"
+                    assert_equal {message chan hello} [$rd1 read]
+                    $rd1 unsubscribe chan
+                } else {
+                    r spublish chan "hello"
+                    assert_equal {smessage chan hello} [$rd1 read]
+                    $rd1 sunsubscribe chan
+                }
+                $rd1 close
```

Comment:
let's add a comment, that this is here just to make sure the defragger didn't break that data structure.